src/components/posts/PostForm/PostForm.tsx

```diff
@@ -21,10 +22,19 @@ import { PostFormFragment } from './PostForm.generated';
 export type PostFormProps<T, R = void> = DialogFormProps<T, R> & {
   /** The pre-existing post to edit */
   post?: PostFormFragment;
+  disableMembership: boolean;
```

Comment:
Let's default this to false. Or if false is the edge case, let's change it to `includeMembership` - still defaulted to false.